TECH.md

```diff
@@ -1,4 +1,5 @@
-OpenAPI Space is a Python API built with Flask and Connexion
- - Flask is a microframework for building Python based web servers. We concluded that it fits well into building a backend where the main function is to provide a REST API
-   - Connexion is a framework used on top of Flask. It allows taking the next step into design first API building, by automatically mapping endpoints when given an API specification in OpenAPI 2.0 format.
- - Integrates into Apinf platform authentication by using the login API of the platform.
+OpenAPI Space is a Python program built with Flask and Connexion
+* The primary objective is to provide access to a Swagger spec database through a REST API.
```

Comment:
Consistent use of OpenAPI / Swagger? OpenAPI definitions or documents should not be referred to as 'specs' to avoid confusion with the OpenAPI specification itself.